src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -95,10 +105,12 @@ private void bindDrivebaseControls() {
 								driveController::getLeftY,
 								driveController::getLeftX,
 								() -> Rotation2d.fromRotations(driveController.getRightX())));
-		driveController.start().onTrue(new InstantCommand(s.drivebaseSubsystem::resetGyro));
```

Comment:
Bad merge? resetGyro needs to be mapped to something

Reply:
Not sure what it should be mapped to since the start is taken up by testing, can make a boolean or something that toggles it

Reply:
Probably should just talk with drive team about what button would work best for `getWithinDistanceTrigger`, or if that's just for testing then take an unused button.

Reply:
okay, will do that in the next set of changes since its just a testing thing

Reply:
Can we add this back and move the get within distance trigger to somewhere else?